types/ember/v2/test/computed.ts

```diff
@@ -146,22 +146,51 @@ const objectWithComputedProperties = Ember.Object.extend({
     setDiff: Ember.computed.setDiff('foo', 'bar'),
     sort1: Ember.computed.sort('foo', 'bar'),
     sort2: Ember.computed.sort('foo', (itemA, itemB) => {
-        if (itemA < itemB) {
-            return -1;
-        } else if (itemA > itemB) {
-            return 1;
-        } else {
-            return 0;
-        }
+        return `${itemA}`.length - `${itemB}`.length;
     }),
     sum: Ember.computed.sum('foo'),
     union: Ember.computed.union('foo', 'bar', 'baz', 'qux'),
     uniq: Ember.computed.uniq('foo'),
     uniqBy: Ember.computed.uniqBy('foo', 'bar'),
-});
-
-const component2 = Component.extend({
-    isAnimal: or('isDog', 'isCat'),
 }).create();
 
-assertType<boolean>(component2.get('isAnimal'));
+// There isn't a good way to assert that something's type is unknown.
+// The best way is to assert that it is Anything, and verify that it gets an error.
+type Anything = {} | undefined | null;
```

Comment:
Interested in getting anyone's thoughts on this. I don't think anything aside from `unknown` should fail to get assigned to this, so `assertType<Anything>(foo); // $ExpectError` should effectively assert that `foo` is `unknown`.